setup.py

```diff
@@ -25,11 +25,11 @@ def read(fname):
     author_email='stefan@foulis.ch',
     packages=find_packages(),
     install_requires=(
-        'Django>=1.5',
-        'easy-thumbnails>=1.0',
-        'django-mptt>=0.6',
-        'django_polymorphic>=0.7',
-        'Unidecode>=0.04',
+        'Django>=1.5,<1.9',
+        'easy-thumbnails>=1.0,<2.4',
+        'django-mptt>=0.6,<0.9',
+        'django_polymorphic>=0.7,<0.9',
```

Comment:
This line https://github.com/divio/django-filer/pull/652/files#diff-c5ee97a47ce37023a9ab5393b05721bfR14

fixing support from 0.8 until 0.9 because there >=0.7,<0.9 and 0.9 not included